controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -328,44 +315,3 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&appsv1.Deployment{}).
 		Complete(r)
 }
-
-// returns a ConfigMap with default settings for rook-ceph operator
-func newRookCephOperatorConfig(namespace string) *corev1.ConfigMap {
-	var defaultCSIToleration = `
-- key: ` + defaults.NodeTolerationKey + `
-  operator: Equal
-  value: "true"
-  effect: NoSchedule`
-
-	config := &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      rookCephOperatorConfigName,
-			Namespace: namespace,
-		},
-	}
-	data := make(map[string]string)
-	data["CSI_PROVISIONER_TOLERATIONS"] = defaultCSIToleration
-	data["CSI_PLUGIN_TOLERATIONS"] = defaultCSIToleration
-	data["CSI_LOG_LEVEL"] = "5"
-	data["CSI_ENABLE_CSIADDONS"] = "true"
-	config.Data = data
-
-	return config
-}
-
-func (r *OCSInitializationReconciler) ensureRookCephOperatorConfig(initialData *ocsv1.OCSInitialization) error {
-	rookCephOperatorConfig := &corev1.ConfigMap{}
-	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: rookCephOperatorConfigName, Namespace: initialData.Namespace}, rookCephOperatorConfig)
-	if err != nil {
-		if errors.IsNotFound(err) {
-			// If it does not exist, create a ConfigMap with default settings
-			return r.Client.Create(context.TODO(), newRookCephOperatorConfig(initialData.Namespace))
```

Comment:
How about creating an empty configmap with ownerRef as it helps in cleanup also? or else we need to document steps for both create and delete with a specific name.

If we create the empty configmap manually we just need to add documentation on what to update/how to update this configmap.

Reply:
Okay, Have to discuss on this if we should keep the empty configmap or not. @jarrpa what do you say?

Reply:
No, I say! :laughing: Having documentation to "update" versus "create" is a trivial and unhelpful distinction. Just don't create it at all, and don't remove it if it exists.

Reply:
@jarrpa If we dont create the configmap, ocs-deployer might fail here https://github.com/red-hat-storage/ocs-osd-deployer/blob/main/controllers/managedocs_controller.go#L1386-L1389 we need a fix here or in ocs-deployer. cc @nb-ohad